app/src/main/java/org/projectbuendia/client/ui/OdkActivityLauncher.java

```diff
@@ -102,6 +132,11 @@ public static void fetchAndShowXform(
         final int requestCode,
         @Nullable final org.odk.collect.android.model.Patient patient,
         @Nullable final Preset fields) {
+        LOG.i("Trying to fetch it from cache.");
```

Comment:
@dancunningham I think what we want is to force-refresh the forms on app-wide sync. It's more bullet-proof than a setting. What do you think?